### PR TITLE
Prime/Echoes Changes

### PR DESCRIPTION
**Pull request by dyceron**

### CODE REVIEW

File: randovania/data/json_data/prime2.txt [hunk 1]
```diff
@@ -2582,7 +2582,11 @@ Asset id: 1115663770
   > Door to Plaza Access
       Trivial
   > Pickup (Energy Tank)
-      Echo Visor and After Agon Energy Returned and Use Screw Attack (Space Jump)
+      All of the following:
+          Echo Visor and After Agon Energy Returned
+          Any of the following:
+              Use Screw Attack (Space Jump)
+              Scan Visor and Extended Dash (Hypermode)
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime2.txt:2589]: No Space Jump?

File: randovania/data/json_data/prime2.txt [hunk 2]
```diff
@@ -9563,8 +9567,13 @@ Asset id: 2722128775
   * Normal Door to Central Area Transport West/Door to Watch Station
   > Pickup (Dark Ammo)
       All of the following:
-          Single Room Out of Bounds (Expert) and Standable Terrain (Expert) and Screw Attack into Tunnels/Openings (Expert) and Use Screw Attack (Space Jump)
-          Boost Ball or Morph Ball Bomb
+          Disabled Room Randomizer
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime2.txt:9570]: No paths to the pickup in case of Room Randomizer

File: randovania/data/json_data/prime2.txt [hunk 3]
```diff
@@ -10029,6 +10038,7 @@ Asset id: 1089946750
               Any of the following:
                   Scan Visor and Combat/Scan Dash (Intermediate)
                   Morph Ball and Roll Jump (Intermediate)
+          Scan Visor and Extended Dash (Hypermode) and Dark World Damage ≥ 50 and Activate Safe Zone
```
Comment (henriquegemignani) [inline randovania/data/json_data/prime2.txt:10041]: Also no SJ?

File: randovania/data/json_data/prime2.txt (hunk 1 above)
Comment (dyceron) [reply randovania/data/json_data/prime2.txt:2589]: Oops

File: randovania/data/json_data/prime2.txt (hunk 2 above)
Comment (dyceron) [reply randovania/data/json_data/prime2.txt:9570]: If it's room rando, youd just get it with spider ball from the other nodes

File: randovania/data/json_data/prime2.txt (hunk 3 above)
Comment (dyceron) [reply randovania/data/json_data/prime2.txt:10041]: ATS extended dash is fine. I forgot SJ for watch station though >_>

**Pull request merged**